src/main/resources/com/google/api/codegen/py/discovery_fragment.snip

```diff
@@ -15,9 +15,12 @@
         1. If not already done, enable the {@serviceTitle}
            and check the quota for your project at
            https://console.developers.google.com/apis/api/{@apiName}
-        2. Install python client library for Google APIs using 'pip install --upgrade google-api-python-client'
-        3. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
-           https://cloud.google.com/sdk/ and run 'gcloud auth application-default login'
+        2. This sample uses Application Default Credentials for authentication. If
+           not already done, install the gcloud CLI from
+           https://cloud.google.com/sdk/ and run
+           'gcloud auth application-default login'
+        3. Install the python client library for Google APIs by running
```

Comment:
In the .snips for the other languages, you don't mention the language name in step 3. Let's be consistent one way or the other. I lean toward including the language name, and capitalizing it: "the Python client library", etc.